### PR TITLE
service: add page

### PR DESCRIPTION
**Pull request by fuerbringer**
Adding the Linux-specific `service` command (used in Gentoo, Debian, etc..).

### CODE REVIEW

File: pages/linux/service.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# service
+
+> Manage services by running init scripts.
+> The full script path should be omitted.
+
+- Start/Stop/Restart/Reload service (start/stop should always be available):
+
+`service {{init_script}} start/stop/restart/reload`
```
Comment (waldyrious) [inline pages/linux/service.md]: please use the `{{start|stop|restart|reload}}` format

File: pages/linux/service.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# service
+
+> Manage services by running init scripts.
+> The full script path should be omitted.
```
Comment (waldyrious) [inline pages/linux/service.md]: What is the assumed path? It would be nice to mention it.

**Comment by waldyrious** [comment]
Thanks for this contribution, looks great already! Just address the inline commends and it should be ready to go.

**Comment by fuerbringer** [comment]
Thank you for the feedback. Made some corrections.

**Comment by waldyrious** [comment]
LGTM :+1:

**Comment by agnivade** [comment]
:+1:

**Pull request merged**